models/articleModels.js

```diff
@@ -0,0 +1,29 @@
+const db = require("../db/connection");
+
+exports.fetchArticleById = (id) => {
+  if (!/\d+/.test(id)) {
+    return Promise.reject({ status: 400, msg: "invalid article_id" });
+  }
+  return db
+    .query(`SELECT * FROM articles WHERE article_id=$1;`, [id])
+    .then(({ rows }) => {
+      if (!rows[0]) {
+        return Promise.reject({ status: 404, msg: "article not found" });
+      }
+      return rows[0];
+    });
+};
+exports.updateArticle = (id, incObj) => {
+  if (Object.keys(incObj).join("") !== "inc_votes") {
```

Comment:
This little bit of error handling is slightly unecessary

If incObj does not have a key of inc_votes then the db.query below will return an error which we can then catch and handle :)